README.md

```diff
@@ -235,6 +235,8 @@ timeout: 600
 
 # The summed length of file paths that can be signed with each signtool call. This parameter should only be relevant if you are signing a large number of files. Increasing the value may result in performance gains at the risk of potentially hitting your system's maximum command length limit. The minimum value is 0 and the maximum value is 30000. A value of 0 means that every file will be signed with an individual call to signtool.
 batch-size: 10000
+
+# A boolean value (true/false) that indicates if the dependencies for this action should be cached by GitHub or not. When using self-hosted runners, caches from workflow runs are stored on GitHub-owned cloud storage. A customer-owned storage solution is only available with GitHub Enterprise Server. When enabled, this option can reduce the duration of the action by at least 1 minute. More info: https://docs.github.com/actions/using-workflows/caching-dependencies-to-speed-up-workflows
 ```
```

Comment:
This is missing the `cache-dependencies` name? And it also not clear that this is enabled by default.

Reply:
This is why I shouldn't make PRs at midnight. Thank you for catching that. 😆